src/applications/personalization/rated-disabilities/actions/index.js

```diff
@@ -8,19 +9,35 @@ export const FETCH_RATED_DISABILITIES_FAILED =
 export const FETCH_TOTAL_RATING_SUCCEEDED = 'FETCH_TOTAL_RATING_SUCCEEDED';
 export const FETCH_TOTAL_RATING_FAILED = 'FETCH_TOTAL_RATING_FAILED';
 
+const serverErrorRegex = /^5\d{2}$/;
+const serviceErrorRegex = /^4\d{2}$/;
+const DISABILITY_PREFIX = 'disability-ratings';
+
 export function fetchRatedDisabilities() {
   return async dispatch => {
     const response = await getData(
       '/disability_compensation_form/rated_disabilities',
     );
 
     if (response.errors) {
-      // record event here
+      const errorCode = response.errors[0].code;
+      if (serverErrorRegex.test(errorCode)) {
+        recordEvent({
+          event: `${DISABILITY_PREFIX}-list-load-failed`,
+          'error-key': '500 internal error',
+        });
+      } else if (serviceErrorRegex.test(errorCode)) {
+        recordEvent({
+          event: `${DISABILITY_PREFIX}-list-load-failed`,
+          'error-key': '401 no disabilities found',
```

Comment:
this is fine as long as you're fine with other 400 errors being reported as 401 errors.

Reply:
Yeah, I was thinking about this. As of right now, we only get 401s back for the disability list which is why this event is worded the way they it is. You're right though, if things change in the future we could certainly tweak the name for this.